src/store/dialogsReducer.js

```diff
@@ -0,0 +1,74 @@
+import { UPDATE_NEW_MESSAGE_BODY, SEND_MESSAGE } from './actionTypes';
+
+const initialState = {
+  dialogs: [
+    {
+      id: '1',
+      name: '105',
+    },
+    {
+      id: '2',
+      name: 'FrinteZz',
+    },
+    {
+      id: '3',
+      name: 'Gaming',
+    },
+    {
+      id: '4',
+      name: 'Pominki',
+    },
+    {
+      id: '5',
+      name: 'Side',
+    },
+    {
+      id: '6',
+      name: 'EFsquad',
+    },
+  ],
+
+  messages: [
+    {
+      id: '1',
+      message: 'qq all',
+    },
+    {
+      id: '2',
+      message: 'hello world',
+    },
+    {
+      id: '3',
+      message: 'priveti4 ot Marmeladi4a',
+    },
+  ],
+
+  newMessageBody: '',
+};
+
+const dialogsReducer = (state = initialState, action) => {
+  switch (action.type) {
+    case UPDATE_NEW_MESSAGE_BODY:
+      state.newMessageBody = action.body;
+      return state;
+
+    case SEND_MESSAGE:
+      const body = state.newMessageBody;
+
+      state.newMessageBody = '';
+      state.messages.push({ id: 8, message: body });
+      return state;
+
+    default:
+      return state;
+  }
+};
+
+const sendMessageCreator = () => ({ type: SEND_MESSAGE });
```

Comment:
Consider moving this function to a separate "action creators" file.

Reply:
/fxd